Xamarin.Forms.Platform.Android/AppCompat/MasterDetailPageRenderer.cs

```diff
@@ -398,6 +398,7 @@ void Update()
 		void UpdateFlowDirection()
 		{
 			this.UpdateFlowDirection(Element);
+			_detailLayout.UpdateFlowDirection();
```

Comment:
Would we want to update the master layout as well? This may not affect the hamburger, but might affect the contents of the master page.

Reply:
On another note, what happens on the other platforms? I see this is just for Android + AppCompat.